mvc/src/main/java/nextstep/mvc/view/JspView.java

```diff
@@ -1,30 +1,47 @@
 package nextstep.mvc.view;
 
+import jakarta.servlet.RequestDispatcher;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
+import java.util.Map;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.Map;
-
 public class JspView implements View {
 
-    private static final Logger log = LoggerFactory.getLogger(JspView.class);
-
     public static final String REDIRECT_PREFIX = "redirect:";
+    private static final Logger log = LoggerFactory.getLogger(JspView.class);
+    private static final String SUFFIX = ".jsp";
+    private final String viewName;
```

Comment:
```suggestion
    private static final String SUFFIX = ".jsp";
    
    private final String viewName;
```

컨벤션에 대한 피드백이라 다소 죄송하지만, 상수와 인스턴스 변수는 개행하는게 어떨까요?